createlinks

```diff
@@ -168,3 +168,15 @@ safe_symlink('/bin/true', 'root/etc/e-smith/events/test-success/S20true');
 safe_symlink('/bin/false', 'root/etc/e-smith/events/test-failure/S20false');
 safe_symlink('/bin/true', 'root/etc/e-smith/validators/test-success/S20true');
 safe_symlink('/bin/false', 'root/etc/e-smith/validators/test-failure/S20false');
+
+#
+# tls-policy-save event
+#
+
+event_templates('tls-policy-save', qw(
+        /etc/systemd/system/cockpit.service.d/nethserver.conf
+));
+
+event_services('tls-policy-save', qw(
+        cockpit restart
```

Comment:
Do not restart Cockpit, otherwise the `tls-policy-save` will be interrupted if called from Cockpit itself.
Cockpit will apply the changes on next restart, 90seconds after the last users has been logged off.

Reply:
how to say to systemctl that the daemon need to be reloaded by `systemctl daemon reload`. It is included by the perl api of nethserver 

It is necessary to add optional settings to systemd

Reply:
The `daemon-reload` is already handled by the runlevel-adjust (or equivalent action).

Reply:
I agree with Giacomo. Do not restart cockpit: it is not a long-running daemon, it stops automatically when the session ends and it is idle for 90 seconds.

Reply:
yep, giacomo show me another unknow way for me with a Env file, I think it will be better

Reply:
The env file is created as a template